src/util/SpawnPed.cpp

```diff
@@ -0,0 +1,50 @@
+#include "core/frontend/Notifications.hpp"
+#include "game/backend/FiberPool.hpp"
+#include "game/backend/ScriptMgr.hpp"
+#include "game/rdr/Natives.hpp"
+
+
+namespace YimMenu
+{
+	bool SpawnPed(std::string model_name, Player player, int playerPed)
+	{
+		FiberPool::Push([=] {
+			Hash model = MISC::GET_HASH_KEY(model_name.c_str());
+			LOG(INFO) << "FOUND HASH";
+			if (STREAMING::IS_MODEL_IN_CDIMAGE(model) && STREAMING::IS_MODEL_VALID(model))
+			{
+				if (!STREAMING::HAS_MODEL_LOADED(model))
+				{
+					STREAMING::REQUEST_MODEL(model, false);
+					ScriptMgr::Yield();
+				}
+
+				Vector3 coords = ENTITY::GET_OFFSET_FROM_ENTITY_IN_WORLD_COORDS(playerPed, 0.0, 3.0, -0.3);
+				auto ped = PED::CREATE_PED(model, coords.x, coords.y, coords.z, static_cast<float>(rand() % 360), 0, 0, 0, 0);
```

Comment:
Why are you randomizing the heading?

Reply:
Just a dynamic element, idk I thought it would be interesting. Should I change it to the players heading?

Reply:
Anything but random